forc/src/utils/parameters.rs

```diff
@@ -0,0 +1,30 @@
+pub const DEFAULT_BYTE_PRICE: u64 = 0;
+pub const DEFAULT_GAS_LIMIT: u64 = fuel_tx::consts::MAX_GAS_PER_TX;
+pub const DEFAULT_GAS_PRICE: u64 = 0;
+
+#[derive(Debug)]
+pub struct TxParameters {
+    pub byte_price: u64,
+    pub gas_limit: u64,
+    pub gas_price: u64,
+}
```

Comment:
I want these to be usable by e.g. `forc deploy` too later. Should they instead be `pub(crate)` if I don't also want to expose them publicly?

Reply:
Fixed in [b93c0da](https://github.com/FuelLabs/sway/pull/1023/commits/b93c0da98ea652440d20b65cd172f1f492745ce7)